conf/app.routes

```diff
@@ -1,4 +1,5 @@
-POST     /business-partner-record             uk.gov.hmrc.cgtpropertydisposals.controllers.BusinessPartnerRecordController.getBusinessPartnerRecord()
-POST     /subscribe                           uk.gov.hmrc.cgtpropertydisposals.controllers.SubscriptionController.subscribe()
-POST     /register-without-id-and-subscribe   uk.gov.hmrc.cgtpropertydisposals.controllers.SubscriptionController.registerWithoutIdAndSubscribe()
-GET      /check-subscription-status           uk.gov.hmrc.cgtpropertydisposals.controllers.SubscriptionController.checkSubscriptionStatus()
\ No newline at end of file
+POST     /business-partner-record                   uk.gov.hmrc.cgtpropertydisposals.controllers.BusinessPartnerRecordController.getBusinessPartnerRecord()
+POST     /subscription                              uk.gov.hmrc.cgtpropertydisposals.controllers.SubscriptionController.subscribe()
+GET      /subscription/:cgtReference                uk.gov.hmrc.cgtpropertydisposals.controllers.SubscriptionController.getSubscription(cgtReference : String)
```

Comment:
I can add a pathbinder for this so it lifts the string to a CgtReference type or shall I leave it?

Reply:
it'd be nice, but not strictly necessary. Since there's not much effort involved let's do it. Here's some code you can nick to help you https://github.com/hmrc/cgt-property-disposals/commit/06e7cb2777e0d237526c7f82b4bb292fd7fc5d49#diff-035875a9790b55d0f6b7c239a0b7b705L27